editor/editor_help.cpp

```diff
@@ -219,21 +226,24 @@ void EditorHelp::_add_type(const String &p_type, const String &p_enum) {
 		if (t.ends_with("[]")) {
 			add_array = true;
 			t = t.replace("[]", "");
+
+			class_desc->push_meta("#Array"); //class
+			class_desc->add_text("Array");
+			class_desc->pop();
+			class_desc->add_text(" [");
```

Comment:
Isn't `Array[T]` (no space) the recommended syntax for typed arrays though? Not sure if typed arrays are already documented anywhere but at least that's the syntax used in #46830 which introduced them (and that's the syntax I'm using).
```suggestion
			class_desc->add_text("[");
```

Reply:
It is, you are correct. I decided to keep the space behind because it makes it easier to read by being less cluttered. 

This is how it looks with one less space, do tell me if you believe the suggested change is still necessary:
![image](https://user-images.githubusercontent.com/66727710/199543850-0099db08-6e3f-40ec-bc0b-6f22ebb4a251.png)

Reply:
>This is how it looks with one less space,

To me it does look better.

>do tell me if you believe the suggested change is still necessary:

Is the change necessary? No, never said it is. But I'd prefer it like that (with no space).